src/Microsoft.DotNet.Build.Tasks/PackageFiles/tool-runtime/project.json

```diff
@@ -28,9 +28,13 @@
         "portable-net45+win8"
       ]
     },
-    "net45": {
+    "net46": {
```

Comment:
If we are going to move to net46 I would like to update the packages and targets that currently reference net45 to be updated to net46 now so we have consistency.